Add Ecovacs event entities (#31968)
home-assistant/home-assistant.github.io: source/_integrations/ecovacs.markdown

- **Button**:
  - `Reset lifespan`: For each supported component, a button entity to reset the lifespan will be created. All disabled by default.
  - `Relocate`: Button entity to trigger manual relocation.
- **Event**:

>   - `Last job`: Provides the stop reason as event_type and additional the room ids, which should be cleaned with the following job.

Reviewer: ```suggestion
  - `Last job`: Provides the stop reason as event_type as well as the IDs of the rooms scheduled to be cleaned in the next job.
```
